packages/ckeditor5-package-generator/lib/templates/ts/package.json

```
@@ -48,10 +48,23 @@
     "@ckeditor/ckeditor5-theme-lark": ">=<%= packageVersions.ckeditor5 %>",
     "@ckeditor/ckeditor5-upload": ">=<%= packageVersions.ckeditor5 %>",
     "@types/chai": "^4.3.1",
+    "@types/ckeditor__ckeditor5-autoformat": "^31.0.1",
```

```suggestion
    "@types/ckeditor__ckeditor5-autoformat": "latest",
```

Let's use the _latest_ version for all typing packages. As we plan to use CKEditor 5 typings, it is acceptable not to specify an exact version.